tests/integration-tests/tests/configure/test_pcluster_configure.py

```diff
@@ -161,3 +238,14 @@ def assert_config_contains_expected_values(
             validator.get("parameter_name")
         )
         assert_that(observed_value).is_equal_to(validator.get("expected_value"))
+
+
+@pytest.fixture()
+def subnet_in_use1_az3(vpc_stack):
+    # use1_az3 does not have c5.xlarge or m6g.xlarge
+    ec2_client = boto3.client("ec2", region_name="us-east-1")
+    subnet_id = ec2_client.create_subnet(
+        AvailabilityZoneId="use1-az3", CidrBlock="192.168.16.0/20", VpcId=vpc_stack.cfn_outputs["VpcId"]
+    )["Subnet"]["SubnetId"]
+    yield subnet_id
+    ec2_client.delete_subnet(SubnetId=subnet_id)
```

Comment:
When does this subnet get deleted? After each individual test finishes or after test _runner finishes?

Reply:
When using fixture, we can use `@pytest.fixture(scope=???)` to specify the scope. The default is the subnet is deleted after each test. I will change it to `@pytest.fixture(scope="module")`

Reply:
Great, was just curious, but I think `module` makes sense here so the same subnet can be used by other tests in this testing file if needed